src/go/rpk/pkg/cli/cmd/root_darwin.go

```diff
@@ -19,4 +20,5 @@ import (
 func addPlatformDependentCmds(
 	fs afero.Fs, mgr config.Manager, cmd *cobra.Command,
 ) {
+	cmd.AddCommand(NewRedpandaDarwinCommand(fs, mgr, redpanda.NewLauncher()))
```

Comment:
redpanda doesn't work on osx, which is the main reason these commands are not installed on darwin at the moment. There are plans to change this such that the commands exist _but_ they exit immediately (depending on the command). I don't think `NewLauncher` is supported here?

Reply:
Removed NewLauncher from the Darwin build.